src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -408,7 +412,7 @@ export default defineComponent({
     },
 
     displayDuration: function () {
-      if (this.useDeArrowTitles && (this.duration === '' || this.duration === '0:00') && this.deArrowCache?.videoDuration) {
+      if (this.showDeArrowTitle && (this.duration === '' || this.duration === '0:00') && this.deArrowCache?.videoDuration) {
         return formatDurationAsTimestamp(this.deArrowCache.videoDuration)
       }
       return this.duration
```

Comment:
With this change the duration would vanish when you click on the `Show Original Details` button, which is undesirable. This change should be reverted to use the DeArrow setting, as this code uses the video duration from the DeArrow API response if it is otherwise unknown (e.g. unwatched videos on the shorts tab on the subscriptions page).